index.html

```diff
@@ -25,16 +25,31 @@
 </head>
 
 <body>
+
   <div class="container paper">
 
+    <div class="paper-navbar">
```

Comment:
The goal of PaperCSS was to create the least classes possible, can you also add styling for `<nav>` tag? This way, people already using `<nav>` tag who include PaperCSS don't have to add any classes to their elements.

Reply:
Ok I will rewrite it little later. Thank you for feedback!
Also, I will remove images in next push.